Docs/general

davidhunter08 opened the pull request:
The animated GIFs used on installation and tutorials docs have been replaced with PNGs.

paul-mitchell-bjss commented:
@neilbmclaughlin Some tests around this functionality would be great. What testing tool were you planning to use?

paul-mitchell-bjss commented:
Closing in preference for #38

davidhunter08 committed:
Installtion screenshots and heroku docs.

Swapped out gifs for pngs, for accessibility reasons.

davidhunter08 committed:
Making pages screenshot images and installtion guide.

Swapped gifs for pngs on making pages.
Added content to requirements page regarding text editor and terminal.

davidhunter08 committed:
Fixed the grammar on installation and tutorial docs.

davidhunter08 committed:
Removed github and azure links from tutorials page.
